react-native-svg: 13.0.0 conflicting  14.0.0

I got this error after installing autocomplete. I did solve the problem using using --legacy-peer-deps but if i need to install other libs this annoy alot when are u going to update react-native-svg to 14.x

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: react-native-autocomplete-dropdown@3.1.0
npm ERR! Found: react-native-svg@14.0.0
npm ERR! node_modules/react-native-svg
npm ERR!   react-native-svg@"^14.0.0" from the root project
npm ERR!   peer react-native-svg@">=5.3" from react-native-feather@1.1.2
npm ERR!   node_modules/react-native-feather
npm ERR!     react-native-feather@"^1.1.2" from react-native-autocomplete-dropdown@3.1.0
npm ERR!     node_modules/react-native-autocomplete-dropdown
npm ERR!       react-native-autocomplete-dropdown@"^3.1.0" from the root project
npm ERR!   1 more (react-native-star-rating-widget)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react-native-svg@"^13.0.0" from react-native-autocomplete-dropdown@3.1.0
npm ERR! node_modules/react-native-autocomplete-dropdown
npm ERR!   react-native-autocomplete-dropdown@"^3.1.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: react-native-svg@13.14.0
npm ERR! node_modules/react-native-svg
npm ERR!   peer react-native-svg@"^13.0.0" from react-native-autocomplete-dropdown@3.1.0
npm ERR!   node_modules/react-native-autocomplete-dropdown
npm ERR!     react-native-autocomplete-dropdown@"^3.1.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

--- a/example/components/ModalExample.tsx
+++ b/example/components/ModalExample.tsx
@@ -10,7 +10,7 @@ export const ModalExample = memo(() => {
       <Button onPress={() => setOpened(prev => !prev)} title="Open modal" />
       <Modal visible={opened}>
         <AutocompleteDropdownContextProvider>
-          <View style={{ paddingHorizontal: 20, flex: 1 }}>
+          <View style={{ paddingHorizontal: 20, flex: 1, paddingTop: 40 }}>
             <RemoteDataSetExample />
             <Button onPress={() => setOpened(false)} title="Close modal" />
           </View>

--- a/example/package.json
+++ b/example/package.json
@@ -1,6 +1,6 @@
 {
   "name": "react-native-autocomplete-dropdown-playground",
-  "version": "3.1.1",
+  "version": "3.1.2",
   "author": "Alexandr Kozhevnikov <onmotion1@gmail.com>",
   "license": "MIT",
   "scripts": {
@@ -14,7 +14,7 @@
     "react": "18.2.0",
     "react-native": "0.72.6",
     "react-native-autocomplete-dropdown": "file:./..",
-    "react-native-svg": "^13.0.0"
+    "react-native-svg": "^14.0.0"
   },
   "devDependencies": {
     "@babel/core": "^7.20.0",

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
     "name": "react-native-autocomplete-dropdown",
-    "version": "3.1.1",
+    "version": "3.1.2",
     "description": "Dropdown Item picker with search and autocomplete (typeahead) functionality for react native",
     "main": "src/index.js",
     "typings": "src/index.d.ts",
@@ -42,7 +42,7 @@
     "peerDependencies": {
         "react": "*",
         "react-native": ">=0.60.0",
-        "react-native-svg": "^13.0.0"
+        "react-native-svg": ">=13.0.0"
     },
     "dependencies": {
         "lodash.debounce": "*",

